core/camel-core-model/src/main/java/org/apache/camel/model/ResumableDefinition.java

```diff
@@ -40,6 +40,11 @@
     @Metadata(required = true, javaType = "org.apache.camel.ResumeStrategy")
     private String resumeStrategy;
 
+    @XmlAttribute(required = true)
```

Comment:
I do not think this should be required = true, as the default value is ERROR

(PS later you use WARN as the default in the resolveLogger)